### DISCUSSION TITLE
Some questions about modeling

### DISCUSSION THREAD

**Discussion by flamewind1** [OpenFAST/openfast#2541] [Q&A]
Dear all.
I am starting a project where I'll have to simulate some real world wind turbines and was wondering if I could get some guidance from you guys. I have a small distributed wind turbine that I would like to simulate in OpenFAST. However, I encountered a problem right at the first step. I have the physical and 3D model of this wind turbine, but the way our wind turbine connects to the hub is quite unique, as shown in the attached image. This is a simplified version of the model. In reality, the blades are connected to the flange with bolts, and the flange is connected to the hub. Therefore, when I import the model data into OpenFAST, where should I start counting the blade root - from the root of the blade or from the hub? Therefore, how can I obtain the following data: BMassDen, FlpStff, EdgStff? Is there any good method? Can these data be exported from SolidWorks?

Thank you for your help.
![2222](https://github.com/user-attachments/assets/70d03f36-2b25-42d1-afc9-dc3158c6ea79)
![1111](https://github.com/user-attachments/assets/68fa2523-fd41-4b33-8507-d8c41be3ed93)

**Comment by jjonkman** [comment]
Dear @flamewind1,

The blade root is typically located at the pitch bearing.  If your wind turbine does not have a pitch bearing, you should choose a location where you believe you should start the aerodynamic and structural dynamic calculations.  In your case, I would guess that location would be the intersection of the cylindrical hub and the cylindrical blade root connection.

If you are unfamiliar with the expressions for `BMassDen`, `FlpStff`, and `EdgStff` in ElastoDyn, see Table 10 from the old FAST v6 User's Guide for their definition: https://openfast.readthedocs.io/en/main/_downloads/d8bd014121d6505cb25cf49bee5eaa80/Old_FAST6_UsersGuide.pdf. I'm not familiar enough with SolidWorks to know if it can output these cross-section beam properties.

Best regards,
